### PR TITLE
Change all checkout command to switch or restore

### LINKED ISSUE #278: Replace checkout with switch/restore
Replace `git checkout` with `git switch` or `git restore` where applicable, as was done in the following pull request: https://github.com/github/training-kit/pull/791

### PR DESCRIPTION
**Pull request by rwnfoo**
Closes #278 

I'm doing committing by file in case we need to revert 😄 May want to squash and merge this PR

### CODE REVIEW

File: docs/11_streamline_workflow_with_aliases.md [hunk 1]
```diff
@@ -44,26 +44,20 @@ Pro Tip # 1:  To edit aliases by hand, you can open the gitconfig file with your
...
+git config --global alias.swc "switch -c"
 ```
 
 Pro Tip # 4: You can create aliases that only call one command.
 
 ```sh
 git config --global alias.s "status -s"
 ```
-
-Pro Tip # 5: Clean up branches quickly and easily.
```
Comment (rwnfoo) [inline docs/11_streamline_workflow_with_aliases.md:65]: This doesn't work for me - maybe someone can test this out

File: docs/11_streamline_workflow_with_aliases.md [hunk 2]
```diff
@@ -44,26 +44,20 @@ Pro Tip # 1:  To edit aliases by hand, you can open the gitconfig file with your
...
-
-```sh
-git config alias.dlb '!git checkout <DEFAULT-BRANCH> && git pull --prune && git branch --merged | grep -v "\*" | xargs -n 1 git branch -d'
```
Comment (rwnfoo) [inline docs/11_streamline_workflow_with_aliases.md:68]: This doesn't work for me - maybe someone can test this out

**Comment by ppremk** [comment]
@rwnfoo @parkerbxyz thank you for 🚀 a little remark.. would be worth to note that the `switch` command is an experimental command and that its behaviour may change in the future as mentioned in the [git-scm](https://git-scm.com/docs/git-switch) doc of the command. 🙂

File: docs/11_streamline_workflow_with_aliases.md (hunk 2 above)
Comment (parkerbxyz) [reply docs/11_streamline_workflow_with_aliases.md:68]: @rwnfoo did you replace `<DEFAULT-BRANCH>` with the name of the default branch in the repo you ran this in?

**Review by Ablatif6c — DISMISSED** [review]

**Review by LAGRANJ — APPROVED** [review]

**Comment by parkerbxyz** [comment]
> @rwnfoo @parkerbxyz thank you for 🚀 a little remark.. would be worth to note that the `switch` command is an experimental command and that its behaviour may change in the future as mentioned in the [git-scm](https://git-scm.com/docs/git-switch) doc of the command. 🙂

Fair point. Thanks, @ppremk. I’m leaning toward proceeding with these changes, despite the commands being experimental. We can revert this PR if these new commands are changed or removed. But if anyone feels strongly that we should wait until they are not considered experimental (I have no idea how long that could be), we can close this PR for now and revisit at that time.

**Review by parkerbxyz — DISMISSED** [review]

**Review by parkerbxyz — APPROVED** [review]

**Comment by chengdagou** [comment]
> Closes #278 
> 
> 
> 
> I'm doing committing by file in case we need to revert 😄 May want to squash and merge this PR

**Review by chengdagou — APPROVED** [review]

File: docs/11_streamline_workflow_with_aliases.md (hunk 2 above)
Comment (parkerbxyz) [reply docs/11_streamline_workflow_with_aliases.md:68]: I’ve opened a PR to restore this pro tip: https://github.com/githubtraining/training-manual/pull/297

**Pull request merged**